### DISCUSSION TITLE
Upgrade Issue - hfs file name

### DISCUSSION THREAD

**Discussion by DS4K** [rejetto/hfs#829] [General]
Hi,

I found when you try to upgrade, if you have previously upgraded it looks like the .exe was renamed to new-hfs.exe. Meaning if you are running the software again while the file is named new-hfs.exe and you try to upgrade it a similar message to the below error will pop up:

EBUSY: resource busy or locked, open 'C:\hfs-windows\new-hfs.exe'

The fix was to stop hfs, rename the file to hfs.exe then run it and upgrade. As the file new-hfs.exe does not exist the error should not occur.

This may have been patched, however I thought I would post incase someone searches that error.

**Comment by rejetto** [comment]
it you got "new-hfs.exe" it means a previous update has stopped before it completed. It's an anomaly.
i'm not surprised it's not working next, but maybe i can do something about it.
thanks for reporting

**Comment by rejetto** [comment]
from next release, the update will work even if you launched it as new-hfs.exe
